### PR TITLE
Stripe force card payment type on Stripe checkout session

### PR DESCRIPTION
**Pull request by PopDaph**
## Description

Forces payment via credit card only for Pro plan

### CODE REVIEW

File: front/lib/plans/stripe.ts [hunk 1]
```diff
@@ -98,6 +98,7 @@ export const createProPlanCheckoutSession = async ({
     mode: "subscription",
     client_reference_id: owner.sId,
     customer_email: user.email,
+    payment_method_types: ["card"],
```
Comment (flvndvd) [inline front/lib/plans/stripe.ts:101]:
```suggestion
    payment_method_types: ["apple-pay", "card", "google-pay"],
```

Comment (flvndvd) [reply front/lib/plans/stripe.ts:101]: (I haven't tested)

Comment (PopDaph) [reply front/lib/plans/stripe.ts:101]: Adding and will test on front-edge

Comment (PopDaph) [reply front/lib/plans/stripe.ts:101]: Stripinou said it will work

Comment (PopDaph) [reply front/lib/plans/stripe.ts:101]: @flvndvd it does not work, I kept card only and deployed to front edge and it still displays the apple pay option from my phone

**Review by flvndvd — APPROVED** [review]

**Pull request merged**